tests/ui/parser/deli-ident-issue-2.stderr

```diff
@@ -1,19 +1,14 @@
-error: mismatched closing delimiter: `]`
-  --> $DIR/deli-ident-issue-2.rs:2:14
-   |
-LL |     if 1 < 2 {
-   |              ^ unclosed delimiter
-LL |         let _a = vec!];
-   |                      ^ mismatched closing delimiter
-
 error: unexpected closing delimiter: `}`
   --> $DIR/deli-ident-issue-2.rs:5:1
    |
+LL | fn main() {
+   |           - the mismatchd closing `}` may be matched here
+LL |     if 1 < 2 {
```

Comment:
I feel like we should still have a label pointing at this opening `{`, stating that the closing `]` matches it.